README.md

```diff
@@ -16,6 +16,25 @@ An [*in*valid](tilt-NOT-valid.json) exemplary document standalone.
 ![](media/dataDisclosed.png)
 
 
+## 📝 Release notes: TILT Version 2.0
+### Data Origin
+According to Art. 20 (1) a "data subject has the right to receive the personal data concerning him or her, which he or she has provided to a controller".
+Because of that, upon a data subject's Right to Data Portability request it is necessary to distinguish between multiple possible data origins. TILT
+Version 2.0 adds an `origin` field to `sources` with three possible options: `received`, `overserved` or `inferred`. This is based on the work of Syrmoudis et al. "Data Portability between Online Services".
+
```

Comment:
remove the "this is based on sentence" since this is quasi "common knowledge". The rest is fine and ready to merge. :)